doc/conf.py

```diff
@@ -162,7 +162,7 @@
 # modindex_common_prefix = []
 
 # A list of warning types to suppress arbitrary warning messages
-suppress_warnings = ["image.not_readable"]
+suppress_warnings = ["image.not_readable", "config.cache"]
```

Comment:
unrelated to the main topic of this PR but is suggested workaround for some new sphinx warning